imgee/storage.py

```diff
@@ -121,7 +125,7 @@ def save_on_s3(filename, remotename='', content_type='', bucket='', folder=''):
         headers = {
             'Cache-Control': 'max-age=31536000',  # 60*60*24*365
             'Content-Type': get_file_type(fp, filename),
-            'Expires': datetime.now() + timedelta(days=365)
+            'Expires': str(datetime.utcnow() + timedelta(days=365))
```

Comment:
If Boto works with datetimes, I'll say just pass that in and don't bother about casting to string.